app/assets/stylesheets/user_new.scss

```diff
@@ -0,0 +1,26 @@
+@import "colors";
+
+#new_user {
+
+  input {
+  background-color: $white;
+  border-color: darken($light-yellow, 5%);
+  border-radius: 5px;
+  color: $slate;
+  font-size: 14pt;
+  margin: 15px 0;
+  }
+
+  input.button{
```

Comment:
Merge rule `input.button` with rule on line 5<br>Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space